python_utils.py

```diff
@@ -310,12 +319,15 @@ def url_open(source_url):
     Returns:
         urlopen. The 'urlopen' object.
     """
+    import certifi
+    import ssl
```

Comment:
Maybe explain why are the imports here and not at the top of the file.

Reply:
I've moved them to the top of the file